src/en-GB/community/events/README.md

```diff
@@ -0,0 +1,76 @@
+<p align="center"><img src="https://i2.wp.com/ceph.io/wp-content/uploads/2016/07/Ceph_Logo_Standard_RGB_120411_fa.png?resize=322%2C148&ssl=1" alt="Ceph" /></p>
+
+# Events
+
+## Frontmatter
+
+```yaml
+---
+title: Event title
+date: yyyy-mm-dd
+end: yyyy-mm-dd
+location: Event location
+venue: Event venue
+map: "https://www.google.com/maps"
+register: "https://www.eventbrite.com/"
+image: "/assets/image.jpg"
+links:
+  videos: "https://www.youtube.com/"
+  slides: "https://ceph.io/event.pdf"
+sponsors:
+  - label: Platinum sponsors
+    list:
+      - name: SoftIron
+        logo: "/assets/image.png"
+        website: "https://softiron.com"
+      - name: RedHat
+        logo: "/assets/image.png"
+        website: "https://redhat.com"
+  - label: Gold sponsors
+    list:
+      - name: SUSE
+        logo: "/assets/image.png"
+        website: "https://suse.com"
+tags:
+  - taxonomy
+  - taxonomy
+---
+
+```
+
+- `title` (String) — Determines the link text as it appears in the event navigation card. Also used as the `<title>` for the page.\*
+- `date` (Date) — Determines the start date of the event.
+- `end` (Date) — Determines the end date of the event.
+- `location` (String) — Determines the location of the event, city and country.
+- `venue` (String) — Determines the venue of the event, building name and street.
+- `map` (String) — Determines the map of the event with a google maps link.\*
+- `register` (String) — Determines the website to register for the event. This is hidden once the end date of the event has pasted.
+- `image` (String) — Determines the asset path for the image shown in the event navigation card and the hero image beneath the title on the event page.
+- `links` (List) - These only appear when the end date of the event has pasted.
+  - `videos` (String) — Determines the website showing videos of the event with a youtube/vimeo link.
+  - `slides` (String) — Determines the website or PDF showing slides of the event.
+- `sponsors` (Array)
+  - `label` (String) — Determines the type of sponsors for the event.
+    `list`
+    - `name` (String) — Determines the name of the sponsor.
+      `logo` (String) — Determines the asset path for the sponsor logo.
+      `website` (String) — Determines the URL of the sponsor website which is applied to the sponsor logo.
```

Comment:
```suggestion
    - `list`
        - `name` (String) — Determines the name of the sponsor.
        - `logo` (String) — Determines the asset path for the sponsor logo.
        - `website` (String) — Determines the URL of the sponsor website which is applied to the sponsor logo.
```

Might just need to tweak the indentent list levels here to ensure the hierarchy is correct.

Reply:
Looks like you maybe seeing different indentation to me. When I look at the README it appears the same as your revised indentation.

```
sponsors:
  - label: Platinum sponsors
    list:
      - name: SoftIron
        logo: "/assets/image.png"
        website: "https://softiron.com"
      - name: RedHat
        logo: "/assets/image.png"
        website: "https://redhat.com"
  - label: Gold sponsors
    list:
      - name: SUSE
        logo: "/assets/image.png"
        website: "https://suse.com"
```